### PR TITLE
Fixed flakiness in cluster size autoscaling e2e

### LINKED ISSUE #14737: Cluster autoscaling e2e tests are flaky
```
Autoscaling [Skipped][Autoscaling Suite] should scale cluster size based on memory utilization

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/autoscaling.go:87
Expected error:
    <*errors.errorString | 0xc2083f3d90>: {
        s: "timeout waiting 20m0s for cluster size to be 3",
    }
    timeout waiting 20m0s for cluster size to be 3
not to have occurred

Autoscaling [Skipped][Autoscaling Suite] should scale cluster size based on cpu utilization

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/autoscaling.go:65
Expected error:
    <*errors.errorString | 0xc2083159d0>: {
        s: "Only 5 pods started out of 6",
    }
    Only 5 pods started out of 6
not to have occurred
```

### PR DESCRIPTION
**Pull request by piosz**
fixes #14737

### CODE REVIEW

File: test/e2e/cluster_size_autoscaling.go [hunk 1]
```diff
@@ -116,6 +121,15 @@ func setUpAutoscaler(metric string, target float64, min, max int) {
 	expectNoError(err, "Output: "+string(out))
 }
 
+func createConsumingRCs(f *Framework, name string, count, cpu, mem int) []*ResourceConsumer {
```
Comment (jszczepkowski) [inline test/e2e/cluster_size_autoscaling.go]: Can you add a comment what are the arguments here? Especially, is cpu & mem per replica or total.

There should be a similar comment in NewStatic(Dynamic)ResourceConsumer. It took me some time to figure out what is going on here.

**Comment by ashcrow** [comment]
Shippable error is not related:

```/root/a4ab7bc9-3a2c-4b5b-8bfb-cd393ea3f6c4.sh: line 96: ./hack/travis/install-etcd.sh: No such file or directory```

File: test/e2e/cluster_size_autoscaling.go (hunk 1 above)
Comment (ashcrow) [inline test/e2e/cluster_size_autoscaling.go]: :+1: I agree with @jszczepkowski

Comment (piosz) [inline test/e2e/cluster_size_autoscaling.go]: Renamed variables

**Comment by piosz** [comment]
PTAL, sorry for delay

**Comment by jszczepkowski** [comment]
LGTM

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Pull request merged**